include/dsn/dist/failure_detector/failure_detector.h

```diff
@@ -173,6 +173,8 @@ class failure_detector :
     bool                 _use_allow_list;
     allow_list           _allow_list;
 
+    ::dsn::rpc_address   _primary_address;
```

Comment:
fixed in https://github.com/qinzuoyan/rDSN/commit/6f6bdfa1b939185f4e55a6252d34e7e092878f8c